src/current/v24.1/read-committed.md

```diff
@@ -405,7 +405,7 @@ To use locking reads:
 
 - If you need to read and later update a row within a transaction, use `SELECT ... FOR UPDATE` to acquire an exclusive lock on the row. This guarantees data integrity between the transaction's read and write operations.
 
-- If you need to read the latest version of a row, but not update the row, use `SELECT ... FOR SHARE` to block all concurrent writes on the row without unnecessarily blocking concurrent reads.
+- If you need to read the latest version of a row, but not update the row, use `SELECT ... FOR SHARE` to acquire a shared lock on the row. This blocks all concurrent writes on the row without unnecessarily blocking concurrent reads.
```

Comment:
Consider adding "concurrent reads or other `SELECT ... FOR SHARE` queries".

Reply:
It might be worth qualifying the first sentence like:

> If you need to read the latest version of a row, and then update a _different_ row within a transaction, use ...